utf8proc-config-version.cmake.cmakein

```diff
@@ -0,0 +1,11 @@
+set(PACKAGE_VERSION @UTF8PROC_FULL_VERSION@)
```

Comment:
Technically, this should probably be the API version (the one in `utfproc.h` — this is the "human-readable" version number) not the ABI version (the one used for versioning the shared-library file).

(Currently, the two version numbers are the same, but it is possible that they may diverge in the future.)